FEM/invariants.h

```diff
@@ -0,0 +1,38 @@
+#ifndef INVARIANTS_H
+#define INVARIANTS_H
+
+#include <cfloat>
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+
+#include "Eigen/Dense"
+
+namespace Math_Group
+{class Matrix;
+}
+
+namespace SolidMath{
+using Math_Group::Matrix;
+
+class Invariants
```

Comment:
actually you don't need to make this as a class. use namespace or make all member functions static const